sample.env

```diff
@@ -111,6 +112,11 @@ CSM_TARGET_SMTP_SERVER_PORT=587
 CSM_TLS_ENABLED=true
 CSM_AUTH_ENABLED=true
 
+# Moove AI feature environment variables
+GOOGLE_ACCESS_KEY_NAME=sample_gcp_service_account.json
+GCP_PROJECT_ID=
```

Comment:
Can we remove the other GCP_PFOJECT_ID (in the jpo_count_metric section)

Reply:
Addressed